packages/contracts/src/resources/site.ts

```diff
@@ -76,23 +280,92 @@ export const siteApi = contract.router(
 				summary: "Gets a list of pubs",
 				description:
 					"Get a list of pubs by ID. This endpoint is used by the PubPub site builder to get a list of pubs.",
-				query: z.object({
+				query: getPubQuerySchema.extend({
+					pubTypeId: pubTypesIdSchema.optional().describe("Filter by pub type ID."),
+					stageId: stagesIdSchema.optional().describe("Filter by stage ID."),
 					limit: z.number().default(10).optional(),
```

Comment:
Hmm I get different results for 
```
curl 'http://localhost:3000/api/v0/c/arcadia-research/site/pubs' --header 'Authorization: Bearer xxxxxxxxxxxxxxxx.00000000-0000-0000-0000-000000000000' | jq 'length'

18
```

```
curl 'http://localhost:3000/api/v0/c/arcadia-research/site/pubs?limit=10' --header 'Authorization: Bearer xxxxxxxxxxxxxxxx.00000000-0000-0000-0000-000000000000' | jq 'length'

10
```

Reply:
I think `limit` can't be `optional` if we want the default to kick in (same for `offset`, I suppose, but that probably matters less since undefined also is 0)

Reply:
ah great catch!

Reply:
i've fixed this!